08_book_titles/book.rb

```diff
@@ -0,0 +1,20 @@
+class Book
+  attr_writer :title
+
+  def title
+    capitalize_title(@title)
+  end
+
+  def capitalize_title(str)
```

Comment:
Hint: there's an cleaner, alternate solution around the corner if you use the variable value where it's due (`%w[and in the of a an].include?(item)`) and then chain functions on `str`.